tests/integration/test_s3_zero_copy_ttl/test_vertical_merge_memory_usage.py

```diff
@@ -1,46 +0,0 @@
-#!/usr/bin/env python3
-import time
-
-import pytest
-from helpers.cluster import ClickHouseCluster
-
-
-single_node_cluster = ClickHouseCluster(__file__)
-small_node = single_node_cluster.add_instance(
-    "small_node",
-    main_configs=["configs/s3.xml"],
-    user_configs=["configs/max_delayed_streams.xml"],
-    with_minio=True,
-)
-
-
-@pytest.fixture(scope="module")
-def started_single_node_cluster():
-    try:
-        single_node_cluster.start()
-
-        yield single_node_cluster
-    finally:
-        single_node_cluster.shutdown()
-
-
-def test_vertical_merge_memory_usage(started_single_node_cluster):
```

Comment:
@alexey-milovidov, this test was mostly fine, it failed only 10 times since `2023-01-01`, including 6 times in one PR https://github.com/ClickHouse/ClickHouse/pull/44869, and it probably was for a reasons, so looks like it helped to find some issue. See [CI DB](https://play.clickhouse.com/play?user=play#d2l0aCAKJyV0ZXN0X3ZlcnRpY2FsX21lcmdlX21lbW9yeV91c2FnZSUnIGFzIG5hbWVfcGF0dGVybiwKJyUlJyBhcyBjb250ZXh0X3BhdHRlcm4sCicyMDIzLTAxLTAxJyBhcyBzdGFydF9kYXRlLAooNDU0NjEsIDQ1MjM2LCA0NDk4NSwgNDc0NzUpIGFzIG5vaXN5X3BycywKKCdTdGF0ZWxlc3MgdGVzdHMgKGFzYW4pJywgJ1N0YXRlbGVzcyB0ZXN0cyAoYWRkcmVzcyknLCAnU3RhdGVsZXNzIHRlc3RzIChhZGRyZXNzLCBhY3Rpb25zKScpIGFzIGJhY2twb3J0X2FuZF9yZWxlYXNlX3NwZWNpZmljX2NoZWNrcwpzZWxlY3QgCnRvU3RhcnRPZkRheShjaGVja19zdGFydF90aW1lKSBhcyBkLApjb3VudCgpLCAgZ3JvdXBVbmlxQXJyYXkocHVsbF9yZXF1ZXN0X251bWJlciksIGFueShyZXBvcnRfdXJsKQpmcm9tIGNoZWNrcyB3aGVyZSBzdGFydF9kYXRlIDw9IGNoZWNrX3N0YXJ0X3RpbWUgYW5kIHB1bGxfcmVxdWVzdF9udW1iZXIgbm90IGluIAooc2VsZWN0IHB1bGxfcmVxdWVzdF9udW1iZXIgYXMgcHJuIGZyb20gY2hlY2tzIHdoZXJlIHBybiE9MCBhbmQgc3RhcnRfZGF0ZSA8PSBjaGVja19zdGFydF90aW1lIGFuZCBjaGVja19uYW1lIGluIGJhY2twb3J0X2FuZF9yZWxlYXNlX3NwZWNpZmljX2NoZWNrcykgCmFuZCB0ZXN0X25hbWUgaWxpa2UgbmFtZV9wYXR0ZXJuIGFuZCAodGVzdF9jb250ZXh0X3JhdyBpbGlrZSBjb250ZXh0X3BhdHRlcm4pCmFuZCBwdWxsX3JlcXVlc3RfbnVtYmVyIG5vdCBpbiBub2lzeV9wcnMgCmFuZCB0ZXN0X3N0YXR1cyBpbiAoJ0ZBSUwnLCAnRkxBS1knLCAnZmFpbHVyZScpIGdyb3VwIGJ5IGQgb3JkZXIgYnkgZCBkZXNj)